downstream/assemblies/platform/assembly-controller-getting-started-credentials.adoc

```diff
@@ -0,0 +1,9 @@
+[id="controller-getting-started-credentials"]
```

Comment:
Can you remove the getting-started(s), identifying the document doesn't make for easy reuse.